CMakeLists.txt

```diff
@@ -105,12 +107,50 @@ else()
   # find_package looks for already-installed system packages.
   # Configure with `-DCMAKE_PREFIX_PATH="/path/to/package1;/path/to/package2"`
   # to add search paths.
+  if (ANDROID)
+    message(STATUS "Please place the version-appropriate SDL3.aar in android-project/app/libs")
+  endif()
   find_package(SDL3 CONFIG REQUIRED)
 
   find_package(iniparser REQUIRED CONFIG COMPONENTS shared)
   target_link_libraries(Isle::iniparser INTERFACE iniparser-shared)
 endif()
 
+# Effectively use as a gate for running the Android build
+if (ANDROID_NDK_PATH)
+
+  FetchContent_GetProperties(SDL3)
+
+  set(SDL_ANDROID_ARCHIVE "${CMAKE_SOURCE_DIR}/android-project/app/libs/SDL3.aar")
+  add_custom_command(OUTPUT "${SDL_ANDROID_ARCHIVE}"
+    COMMAND ${Python3_EXECUTABLE} "${SDL3_SOURCE_DIR}/build-scripts/build-release.py"
+       --actions android
+       --fast
+       --android-ndk-home=${ANDROID_NDK_PATH}
+       --force
+       --root="${SDL3_SOURCE_DIR}"
```

Comment:
The `build-release.py` script is not really meant to be used like this.
There aren't any promises about argument stability.

Reply:
The scripts in `SDL/build-scripts` are internal-only.

I don't know how, but is it possible to let Gradle know that the cmake build step will make available java sources somewhere, which Gradle then can compile?

Reply:
> I've moved the .aar building to gradle but i didn't remove the use of the SDL provided build script. Using the script is the only realistic option as SDL has no gradle target to build the aar and recreating its functionality will bring more headaches than not. If the script changes and breaks the build then it can be fixed.

That's reasonable and acceptable. SDL's gradle script is lacking in its aar-building capabilities.
It was [capable to create a aar until this PR](https://github.com/libsdl-org/SDL/pull/10450), but the aar was missing features that developers expect from a SDL3 archive, so we removed it.